{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 4, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_emptyonlyonefield", "rerun": "0"}, "1": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_backtosigninpage", "rerun": "0"}, "2": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_loginsuccessfully", "rerun": "0"}, "3": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_SQLInjection", "rerun": "0"}}, "suite_name": "Login.py"}, "1": {"suite_name": "SignUp.py", "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_not10characternumber", "rerun": "0"}, "1": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_backtologinpage", "rerun": "0"}, "2": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_wrongEmailId", "rerun": "0"}, "3": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_sameusername", "rerun": "0"}, "4": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_passwordsAreNotSame", "rerun": "0"}, "5": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_emptyonlyonefield", "rerun": "0"}, "6": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_emptyalltheinputs", "rerun": "0"}, "7": {"status": "FAIL", "message": "   selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n     (Session info: chrome=91.0.4472.106)\n", "test_name": "test_signinsuccessfully", "rerun": "0"}}, "status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 8, "total_error": 0}}}}, "date": "June 18, 2021", "start_time": 1624040060.8382044, "total_suite": 2, "status": "FAIL", "status_list": {"pass": "0", "fail": "12", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "12"}